safe/report/test/test_impact_report.py

```diff
@@ -102,13 +102,15 @@ def setUp(self):
         # Change displacement rate so the result is easily distinguished
         self.default_displacement_rate = flood_hazard_classes['classes'][0][
             'displacement_rate']
-        flood_hazard_classes['classes'][0][
-            'displacement_rate'] = self.custom_displacement_rate
+        # flood_hazard_classes['classes'][0][
+        #     'displacement_rate'] = self.custom_displacement_rate
```

Comment:
Any reason to keep oldcode? It's in a test file, it's ok, but I usually remove code which will be dead forever